src/cdfmean.f90

```diff
@@ -115,10 +124,15 @@ PROGRAM cdfmean
      PRINT *,'                   partial steps.'
      PRINT *,'       [ -var ]  : also compute the spatial variance of cdfvar '
      PRINT *,'       [ -zeromean ] : create a file with cdfvar having a zero spatial mean.'
+     PRINT *,'       [ -out_suff OUT-file-suffix ] : specify SUFFIX (+ directory) of the output files'
+     PRINT *,'       [ -basins n_basins basin_names.. ] : specify basins in ',TRIM(cn_fbasins),' to be used'
```

Comment:
Does the NEMO basin file have an accepted convention in this respect? I'm not aware of an 'official' tool to produce such a file, so are "tmaskatl", "tmaskind" etc accepted standard names in the community? Are they used in NEMO?

I ask also because it may be an idea in the future to extend freely-defined basin mask specification to replace hard-coded basin mask names in some tools (e.g. `cdfmoc`). The mask itself can vary so "tmaskatl" is not in itself a standard, even if the name is.